src/plugin/commands.sjs

```diff
@@ -111,9 +117,31 @@ module.exports = {
                         } else {
                             plugin = "UnknownSource";
                         }
-
+                        
+                        if (globalIgnoreItems.indexOf(commandName) !== -1) {
+                            client.note("PluginCommands", format("Ignoring '%s:%s (global)'.", plugin, commandName));
+                            return;
+                        }
+                        
+                        for(var i = 0; i < ignoreCommandInSpecificPluginList.length; i++)
```

Comment:
You want `ignoreCommandInSpecificPluginList.some(...), not a for loop. For loops are against the style of the codebase.